tsl/src/bgw_policy/drop_chunks_api.c

```diff
@@ -55,6 +58,137 @@
 										  Int32GetDatum(0),                                        \
 										  Float8GetDatum(0)))
 
+/* TODO:: Q:: These should probably be in its own file? */
+Datum
+ts_integer_from_now_func_get_datum(int64 interval, Oid time_dim_type, Oid now_func)
+{
+	Datum now;
+
+	AssertArg(IS_INTEGER_TYPE(time_dim_type));
+
+	ts_integer_now_func_validate(now_func, time_dim_type);
+
+	// bgw luncher starts a transactions so here I can assume I am always inside a transaction
+	// right? no need for StartTransactionCommand();
```

Comment:
bgw luncher starts a transaction so here I can assume I am always inside a transaction right? so no need for StartTransactionCommand();